src/parse.rs

```diff
@@ -1013,7 +1013,10 @@ impl InlineStack {
         count: usize,
         both: bool,
     ) -> Option<InlineEl> {
-        let lowerbound = min(self.stack.len(), self.get_lowerbound(c, count, both));
+        if self.stack.is_empty() {
```

Comment:
This doesn't seem right, and it feels like it might be possible to construct failing cases. One is that the extra lookback in the stack needs more than 1. Another is that it fails to call `set_lowerbound()` in the case that the stack is empty.